data/account/participationRegistry.go

```diff
@@ -82,20 +83,40 @@ type ParticipationRecord struct {
 	Voting *crypto.OneTimeSignatureSecrets
 }
 
-// StateProofKey is a placeholder for the real state proof key type.
-// PKI TODO: Replace this with a real object.
-type StateProofKey []byte
+// StateProofKey defined the type used for the compact certificate signing key
+type StateProofKey crypto.GenericSigningKey
 
 // ParticipationRecordForRound adds in the per-round state proof key.
 type ParticipationRecordForRound struct {
 	ParticipationRecord
 
-	StateProof StateProofKey
+	StateProofSecrets *merklekeystore.Signer
 }
 
 // IsZero returns true if the object contains zero values.
 func (r ParticipationRecordForRound) IsZero() bool {
-	return r.StateProof == nil && r.ParticipationRecord.IsZero()
+	return r.StateProofSecrets == nil && r.ParticipationRecord.IsZero()
+}
+
+// VotingSigner returns the voting secrets associated with this Participation account,
+// together with the KeyDilution value.
+func (r *ParticipationRecordForRound) VotingSigner() crypto.OneTimeSigner {
```

Comment:
can we remove `participation.VotingSigner`?

Reply:
It is used by:
`asyncPseudonode.makeVotes
asyncPseudonode.makeProposals
AlgorandFullNode.MakePrioResponse`,
as I replaced the `Participation` type used there.

Reply:
you are right
so remove  `Participation.VotingSigner`